### DISCUSSION TITLE
add "only pen" support

### DISCUSSION THREAD

**Discussion by Wladefant** [personalizedrefrigerator/js-draw#48] [Ideas & Feature Requests]
remove the finger paint support and only paint with a pen. Or put it in settings where you can choose between pen support OR finger and Pen support.

Right now even while I am drawing I often draw with my skin

So I want only the option to draw with a pen. When doing something with a finger, for example the option of moving an object could be added

**Comment by personalizedrefrigerator** [comment]
See https://github.com/personalizedrefrigerator/js-draw/discussions/46#discussioncomment-7393900
